hack/test-cmd.sh

```diff
@@ -67,6 +68,7 @@ export ETCD_PEER_PORT=${ETCD_PEER_PORT:-27001}
 setup_env_vars
 export SUDO=''
 mkdir -p "${ETCD_DATA_DIR}" "${VOLUME_DIR}" "${FAKE_HOME_DIR}" "${MASTER_CONFIG_DIR}" "${NODE_CONFIG_DIR}" "${LOG_DIR}"
+reset_tmp_dir
```

Comment:
> It would be nice if we were passing arguments to these functions instead of declaring globals that are used inside of them. Not sure if this is the pull to do that in.

It's not.  There's a also a choice to be made regarding these.  These functions are here to make the 80% case easier to deal with.  Standardize the env vars in our scripts and our usage allowed for tremendous simplification.

If this were product code, I'd agree with you on this point, but the main purpose here is to keep the test scripts short, sweet, and predictable in the sense that everything works the same way with the same names.